src/Interaction/Comment/Infrastructure/MongoDBCommentRepository.py

```diff
@@ -42,19 +36,16 @@ def add_comment(
         comment_dict = comment.dict()
         comment_dict["_id"] = ObjectId()
         comment_dict["user_id"] = ObjectId(comment_dict["user_id"])
```

Comment:
add verification if the publication exist before add the comment (publication access logic should be at the application layer)